automatic-app-upgrades/README.md

```diff
@@ -0,0 +1,37 @@
+# Automatic App upgrades
+
+## Solution proposal
+
+We propose to use OCI registries, Flux controllers, and GitHub repositories to fully automate App upgrades.
+
+## Motivation
+
+We would like to automate App upgrades. Since CAPI clusters will be packaged as Apps as well, this would, by extension, allow us to automate their upgrades as well. This would serve our customers, as well as unblock some of the epics and stories, like pull-based app collections.
+
+## Solution in detail
+
+Use OCI registries to hold Helm charts. Use GitOps, in particular Flux's Image Automation Controllers, to monitor the registry and upgrade apps when a newer version of the chart is pushed.
+
+Since version 3.8.0, Helm [enables storing charts in OCI registries](https://helm.sh/blog/storing-charts-in-oci/). OCI registries, or Open Containers Initiative registries, allow storing arbitrary artifacts other than containers in them using a common storage standard.
+
+One of the registries that are fully conformant with the OCI specification is Azure Container Registry. The registry we currently use as a mirror or backup to our other registries.
```

Comment:
Let's explicitly state that this would also mean we use ACR in a first implementation iteration. It's just kind of implied here :)